ng-dev/BUILD.bazel

```diff
@@ -9,6 +9,11 @@ NG_DEV_EXTERNALS = [
     # Package uses `__filename` and `__dirname` and cannot be bundled in ESM. We do not
     # intend to provide interop globals for this as it could hide other significant issues.
     "@yarnpkg/lockfile",
+    # SPDX uses dynamic imports that work at runtime, but esbuild complains about.
+    "spdx-ranges",
+    "spdx-license-ids",
+    "spdx-license-ids/deprecated",
+    "spdx-exceptions",
```

Comment:
You should be able to just add `spdx-satisfies` instead of manual transitive deps. Do you also recall how ESBuild complained OOC.

Reply:
I tried, but it continues to complain about them.

The error is 
```
✘ [ERROR] Could not resolve "spdx-license-ids"

    node_modules/spdx-correct/index.js:17:29:
      17 │ var spdxLicenseIds = require('spdx-license-ids')
         ╵                              ~~~~~~~~~~~~~~~~~~
  You can mark the path "spdx-license-ids" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.
```

Same thing for all of the other ones.

Reply:
Seems to be because e.g. https://unpkg.com/browse/spdx-license-ids@3.0.11/ is just exposing a JSON file. We would need to tell ESBuild to also load JSON files (which is an option IIRC).

Reply:
From looking at the docs, it looks like the json file loader is included by default.

Reply:
Yeah, but we aren't setting it up. https://github.com/angular/dev-infra/blob/main/bazel/esbuild/index.bzl#L31

I think we can add `.json` here (and in other places)

Reply:
Ahh that did the trick.